src/module_1/module_1_meteo_api.py

```diff
@@ -88,81 +111,96 @@ def clean_data(raw_data):
                 cleaned_data[key] = value
     return cleaned_data
 
+
 def process_data(data):
+    """
+    Reads a df and for each city calculates mean + std for each variable
+    """
+    calculated_df = data[["city", "time"]].copy()
+    for var in VARIABLES.split(","):
```

Comment:
You don't need the split if you follow my above suggestion

Reply:
Solved